packages/core/src/state/__tests__/applyUpdatersToResults.ts

```diff
@@ -7,12 +7,6 @@ describe('applyUpdatersToResults', () => {
     [ArticleResourceWithOtherListUrl.listShape().getFetchKey({})]: ['1', '2'],
   };
 
-  it('returns results if updaters is not defined', () => {
```

Comment:
does this indicate a breaking change?

Reply:
No this checked was hoisted to the reducer. This function is not exported from module